cmd/kubeadm/app/constants/constants.go

```diff
@@ -31,6 +31,9 @@ const (
 	APIServerKubeletClientCertName           = "apiserver-kubelet-client.crt"
 	APIServerKubeletClientKeyName            = "apiserver-kubelet-client.key"
 
+	ServiceAccountKeyBaseName = "sa"
+	ServiceAccountKeyName     = "sa.key"
```

Comment:
Only the controller manager needs the private key. The apiserver just needs the public portion to verify. It will extract the public key from the private key if that's what you give it, but if you are setting up a key for this purpose, splitting it into a public/private pair makes it possible to limit the audience for the private key, which is nice.